src/sources/languages_2.ts

```diff
@@ -0,0 +1,28 @@
+import { BrowserEngineKind } from '../types'
```

Comment:
I think it would be better to get rid of the old version of the signal and use the new implementation in the detector